packages/storybook/stories/progress-indicator.stories.tsx

```diff
@@ -2,7 +2,7 @@ import { ProgressIndicator } from '@equisoft/design-elements-react';
 import { StoryFn as Story } from '@storybook/react';
 
 export default {
-    title: 'Components/Progress Indicator',
+    title: 'Deprecated/Progress Line (deprecated)',
```

Comment:
on va également changer le nom du composant. Ca va créer un breaking change mais ce n'est pas that bad puisqu'il est seulement utilisé dans le comparateur financier. J'en ai jasé à Max et il est bien parfait avec ça.

Reply:
Parfait. Est-ce que je dois modifier quelque chose dans cette PR ou c'est ok?

Reply:
@Genmoree au final, on va le supprimer et il ne se retrouvera pas dans une section deprecated.